hmt_escrow/job.py

```diff
@@ -1283,4 +1360,4 @@ def _create_escrow(self, gas: int = GAS_LIMIT) -> bool:
 if __name__ == "__main__":
     import doctest
     from test_manifest import manifest
-    doctest.testmod(raise_on_error=True)
```

Comment:
this is what failed CI before, IIRC.  If this is not set it may fail and not `exit 1` thus failing a build

Reply:
Ah yes, always locally disabling to see track traces.

Reply:
Can you also throw a comment in there to remind ppl?  I randomly remembered this and still not 100% sure on it